Assets/MRTK/SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2373,7 +2393,7 @@ private void ScaleHandle(Handle handle, bool lerp = false)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         private bool IsPointWithinBounds(Vector3 point, float radiusSqr)
         {
-            return (transform.position - point).sqrMagnitude < radiusSqr;
+            return (Vector3.Scale(TargetBounds.center, TargetBounds.gameObject.transform.lossyScale) + transform.position - point).sqrMagnitude < radiusSqr;
```

Comment:
is targetbounds center relative to the transform of the component? or is this returning world space coordinates?

Reply:
I was copying what we did in bounds control 

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/18fe32fd8d627216754885c7fecf4cf2434aa07d/Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/ProximityEffect/ProximityEffect.cs#L153

Reply:
I wasn't sure if the center of those bounds is in world or local space - makes sense in local space, closing this issue